CHANGELOG.md

```diff
@@ -36,6 +36,7 @@ CHANGELOG
 - Upgrade Slurm to version 22.05.5.
 - Upgrade EFA installer to version 1.18.0.
 - Upgrade NICE DCV to version 2022.1-13300.
+- Allow for suppressing the SingleSubnetValidator for Queues
```

Comment:
Missing final period.

Reply:
We should use backticks for both SingleSubnetValidator and Queues words.

Reply:
got it ty!